test/visibility/import/deeperNesting-parentImport-unqualified.chpl

```diff
@@ -7,7 +7,7 @@ module A {
     var x: int;
 
     module C {
-      import A.B;
+      import A.B.foo;
 
       proc main() {
         foo(); // Should work, foo is still in scope to main()
```

Comment:
I think this test should be split into a version that generates an error (without the update you made) and this version.  In both cases, this comment should be updated to reflect the new status quo

Reply:
See https://github.com/chapel-lang/chapel/pull/15312/commits/833d0f1d15a95c8d25be98433011f5cb0f768ddb